core/lib/search/adapter/api/searchAdapter.ts

```diff
@@ -75,14 +76,36 @@ export class SearchAdapter {
     url = url + '/' + encodeURIComponent(Request.index) + '/' + encodeURIComponent(Request.type) + '/_search'
     url = url + '?' + queryString.stringify(httpQuery)
 
+    let queryMethod = config.elasticsearch.queryMethod
+    if (Request.type === 'product' && Request.sort) {
```

Comment:
As `asc` sorting is default one. You should make support for this in these 2 lines (80-81)